src/include/miopen/fusion.hpp

```diff
@@ -33,16 +33,22 @@
 #include <miopen/solver.hpp>
 #include <miopen/op_kernel_args.hpp>
 #include <miopen/fusion_ops.hpp>
+#include <miopen/fusion/fusion_invoke_params.hpp>
 #include <miopen/activ.hpp>
 
 #include <set>
 #include <vector>
 #include <unordered_map>
 
+#define MIOPEN_CHECK(x)          \
```

Comment:
@JehandadKhan Is this macro used anywhere? If so, let's move it to some common header.

```suggestion
#define MIOPEN_CHECK_STATUS(x)          \
```

Reply:
This macro is only used in `fusion.cpp` to construct fusion plans to support dedicated CBA API. I have moved it back to the CPP file since its not used anywhere else.